erasure_code/ec_highlevel_func.c

```diff
@@ -30,6 +30,7 @@
 #include "erasure_code.h"
 #include "types.h"
 
+#if __x86_64__  || __i386__
```

Comment:
These predefines are not in cl so this fails the windows build.  Perhaps change to:
`#if __x86_64__  || __i386__ || _M_X64 || _M_IX86`